lib/screens/creator_hub/creator_hub_view_model.dart

```diff
@@ -48,46 +48,24 @@ class CreatorHubViewModel extends ChangeNotifier {
     notifyListeners();
   }
 
-  Future<void> saveNft(File? file, EaselProvider provider) async {
-    final loading = Loading().showLoading(message: "uploading".tr());
-    provider.initializeTextEditingControllerWithEmptyValues();
-    final uploadResponse = await remoteDataSource.uploadFile(file!);
+  Future<void> updateNft() async {
+    final loading = Loading().showLoading(message: "loading ...");
+    nftList = await localDataSource.getNfts();
```

Comment:
Remove access to local data store and remote datastore. Give access of repository only.